corehq/apps/couch_sql_migration/tests/test_migration.py

```diff
@@ -781,17 +781,17 @@ def test_v1_case(self):
         self._compare_diffs([])
 
     def test_timings(self):
-        with patch_datadog() as received_stats:
+        with capture_metrics() as received_stats:
             self._do_migration_and_assert_flags(self.domain_name)
         tracked_stats = [
-            'commcare.couch_sql_migration.unprocessed_cases.count.duration:',
-            'commcare.couch_sql_migration.main_forms.count.duration:',
-            'commcare.couch_sql_migration.unprocessed_forms.count.duration:',
-            'commcare.couch_sql_migration.count.duration:',
+            'commcare.couch_sql_migration.unprocessed_cases.count',
+            'commcare.couch_sql_migration.main_forms.count',
+            'commcare.couch_sql_migration.unprocessed_forms.count',
+            'commcare.couch_sql_migration.count',
```

Comment:
@millerdev FYI this no longer checks for the duration tag, only the metric name. Let me know if you think that's an issue.